### PR TITLE
Default to ckan:2.4 image

### PR DESCRIPTION
**Pull request by mchelen**
ckan:2.3 has a critical error that is resolved in 2.4 https://github.com/datacats/datacats/issues/334

### CODE REVIEW

File: datacats/cli/pull.py [hunk 1]
```diff
@@ -12,15 +12,14 @@
...
+    'datacats/ckan:2.4',
     'datacats/postgres',
     'datacats/solr'
     ]
 
 EXTRA_IMAGES = [
     'datacats/lessc',
-    'datacats/ckan:latest',
-    'datacats/ckan:2.4'
+    'datacats/ckan:latest'
```
Comment (wardi) [inline datacats/cli/pull.py]: shouldn't this have a `datacats/ckan:2.3' added?

Comment (mchelen) [inline datacats/cli/pull.py]: I wasn't sure if there was any reason to have the 2.3 version, but I added it back in case anyone needs it.

**Comment by wardi** [comment]
https://github.com/datacats/datacats/blob/master/datacats/cli/create.py#L33 will need to be changed too or this won't change the default

**Comment by mchelen** [comment]
@wardi Ah ok, added to the PR. Does that line change the default value or is it just part of the documentation?

**Comment by wardi** [comment]
@mchelen it does both! docopt is magic (in a good way)

@JackMc it looks like this should work, If you're happy with it please go ahead and merge

**Pull request merged**